### PR TITLE
Make TURN server port range configurable

### LINKED ISSUE #32: Reduce / make configurable the range of needed udp ports
Some appliances restrict the range of ports in single a port forwarding rule. This makes difficilt/unpractical to comply with the required very large udp port range.

It would be nice to reduce that range and/or configure it basing on real needs (e.g. max envisioned number of contemporay a/v calls.

### PR DESCRIPTION
**Pull request by horazont**
Fixes #32; see there and in the text for details and rationale.

### CODE REVIEW

File: ansible/files/bin/start-coturn.sh [hunk 1]
```diff
@@ -16,7 +16,11 @@ done
 
 TURN_EXTERNAL_IP="$(snikket-turn-addresses "$SNIKKET_DOMAIN")"
 
+min_port="${SNIKKET_TWEAK_TURNSERVER_MIN_PORT:-49152}"
+max_port="${SNIKKET_TWEAK_TURNSERVER_MIN_PORT:-65535}"
```
Comment (mwild1) [inline ansible/files/bin/start-coturn.sh:20]: I think this should be `SNIKKET_TWEAK_TURNSERVER_MAX_PORT` ?

File: docs/advanced/firewall.md [hunk 2]
```diff
@@ -21,6 +21,27 @@ Snikket currently requires the following ports to be open/forwarded:
...
- | :------------ | :----------------------------------------------------------------------------------|
- | 49152-65535   | Audio/Video Data Proxy (Turn Data)                                                 |
+ |**UDP only**  |                                                                                    |
+ | :----------- | :----------------------------------------------------------------------------------|
+ | 49152-65535  | Audio/Video Data Proxy (Turn Data, see below)                                      |
+
+
+## Changing the turnserver port range
+
+The STUN/TURN server is required for audio/video (A/V) calls to work reliably on all kinds of "difficult" client networks. For this, a relay connection is established which routes the (encrypted) A/V data via your Snikket server. As generally the number of concurrent calls is not known and it needs to compete with ports already in use on the machine, the TURN server defaults to a range with a high number of ports (about 16 thousand).
```
Comment (mwild1) [inline docs/advanced/firewall.md:31]: It's implied, but not explicitly stated, that more ports == more concurrent calls. I think it would be good to clarify that a smaller range means fewer concurrent calls (do we know roughly how many?). It's often asked why so many ports are needed.

Maybe worth an FAQ entry as well actually...

**Review by mwild1 — CHANGES_REQUESTED** [review]

File: docs/advanced/firewall.md (hunk 2 above)
Comment (horazont) [reply docs/advanced/firewall.md]: Thanks, I added some text. Might be too much text, or too technical, or not technical enough. Interested in more feedback.

File: ansible/files/bin/start-coturn.sh (hunk 1 above)
Comment (horazont) [reply ansible/files/bin/start-coturn.sh]: Meh. Fixed.

File: docs/advanced/firewall.md (hunk 2 above)
Comment (mwild1) [reply docs/advanced/firewall.md]: I think it's about right. Possibly a bit wordy for this document, but what I will probably do is borrow some of the text for the FAQ entry, and link to that from here or something (or maybe the other way around... hmm). LGTM for now though.

**Review by mwild1 — APPROVED** [review]
